### PR TITLE
Fix a logic bug on parentless spans

### LINKED ISSUE #771: Bug in RichConsoleSpanExporter
**Describe your environment**
Python 3.9.7, Snippet from `pipenv graph`:
```
opentelemetry-exporter-richconsole==0.25b2
  - opentelemetry-api [required: ~=1.3, installed: 1.10a0]
  - opentelemetry-sdk [required: ~=1.3, installed: 1.10a0]
    - opentelemetry-api [required: ==1.10a0, installed: 1.10a0]
  - opentelemetry-semantic-conventions [required: ==0.25b2, installed: 0.25b2]
  - rich [required: >=10.0.0, installed: 10.12.0]
    - colorama [required: >=0.4.0,<0.5.0, installed: 0.4.4]
    - commonmark [required: >=0.9.0,<0.10.0, installed: 0.9.1]
    - pygments [required: >=2.6.0,<3.0.0, installed: 2.10.0]
```

**Steps to reproduce**
Given this code:
```
from opentelemetry import trace
from opentelemetry.sdk.resources import SERVICE_NAME, Resource
from opentelemetry.exporter.richconsole import RichConsoleSpanExporter
from opentelemetry.sdk.trace import TracerProvider
from opentelemetry.sdk.trace.export import BatchSpanProcessor
from opentelemetry.sdk.trace.export import ConsoleSpanExporter

APP_SERVICE_NAME = "fastapi-goofing"

trace.set_tracer_provider(
    TracerProvider(
        resource=Resource.create({SERVICE_NAME: APP_SERVICE_NAME})
    )
)

rich_console_exporter = RichConsoleSpanExporter(
    service_name=APP_SERVICE_NAME,
)


console_exporter = ConsoleSpanExporter(
    service_name=APP_SERVICE_NAME
)

trace.get_tracer_provider().add_span_processor(
    BatchSpanProcessor(rich_console_exporter)
    #BatchSpanProcessor(console_exporter)
)
tracer = trace.get_tracer(__name__)

with tracer.start_as_current_span("foo"):
    with tracer.start_as_current_span("bar"):
        with tracer.start_as_current_span("baz"):
            print("Hello world from OpenTelemetry Python!")


```
The `RichConsoleSpanExporter` throws this error:
```
Exception while exporting Span batch.
Traceback (most recent call last):
  File "/home/trond/Documents/projects/fastapi-goofring/.venv/lib/python3.9/site-packages/opentelemetry/sdk/trace/export/__init__.py", line 331, in _export_batch
    self.span_exporter.export(self.spans_list[:idx])  # type: ignore
  File "/home/trond/Documents/projects/fastapi-goofring/.venv/lib/python3.9/site-packages/opentelemetry/exporter/richconsole/__init__.py", line 166, in export
    child = parents[span.parent.span_id].add(
AttributeError: 'NoneType' object has no attribute 'span_id'
```
If I replace the Rich exporter with the regular Console exporter, everything runs nicely without problems

**What is the expected behavior?**
Code runs without exceptions

**What is the actual behavior?**
Exception is thrown

**Additional context**

### PR DESCRIPTION
**Pull request by tonybaloney**
Fixes #771

### CODE REVIEW

File: exporter/opentelemetry-exporter-richconsole/src/opentelemetry/exporter/richconsole/__init__.py [hunk 1]
```diff
@@ -154,18 +154,26 @@ def export(self, spans: typing.Sequence[ReadableSpan]) -> SpanExportResult:
             _child_to_tree(child, span)
 
         for span in spans:
-            if span.parent and span.parent.span_id not in parents:
-                child = tree.add(
-                    label=Text.from_markup(
-                        f"[blue][{_ns_to_time(span.start_time)}][/blue] [bold]{span.name}[/bold], span {opentelemetry.trace.format_span_id(span.context.span_id)}"
+            if span.parent:
```
[the remark this answers is not in the document]
Comment (tonybaloney) [reply exporter/opentelemetry-exporter-richconsole/src/opentelemetry/exporter/richconsole/__init__.py]: You're right that makes more sense.

**Comment by tonybaloney** [comment]
Added a simple test, any pointers for useful test examples would be welcome. I looked at data dog, but its really complicated!

**Review by owais — APPROVED** [review]

**Comment by owais** [comment]
@tonybaloney please add a changelog entry

**Comment by tonybaloney** [comment]
> @tonybaloney please add a changelog entry

done

**Pull request merged**